code/datums/status_effects/debuffs.dm

```diff
@@ -80,6 +80,45 @@
 	desc = "You've fallen asleep. Wait a bit and you should wake up. Unless you don't, considering how helpless you are."
 	icon_state = "asleep"
 
+//STASIS
+/datum/status_effect/incapacitating/stasis
+        id = "stasis"
+        duration = -1
+        tick_interval = 10
+        alert_type = /obj/screen/alert/status_effect/stasis
+        var/last_dead_time
+
+/datum/status_effect/incapacitating/stasis/proc/update_time_of_death()
+        if(last_dead_time)
+                var/delta = world.time - last_dead_time
+                var/new_timeofdeath = owner.timeofdeath + delta
+                owner.timeofdeath = new_timeofdeath
+                owner.tod = station_time_timestamp(wtime=new_timeofdeath)
```

Comment:
```suggestion
                owner.tod = WTIME_TO_STATION_TIME_TIMESTAMP("hh:mm:ss", new_timeofdeath)
```

Reply:
This is throwing an issue, doesn't seem to be recognizing all the vars

Reply:
fixed it